assets/setup-wizard/data/selectors.js

```diff
@@ -1,8 +1,3 @@
-/**
- * Internal dependencies
- */
-import { steps } from '../steps';
```

Comment:
Customization that added the `isNext` and `isComplete` through the `getNavigationSteps` selector is not needed anymore based on the new design and behavior.

Maybe later we will want to add something for the user to return to the step where the user stopped (in case the close the setup wizard before completing it), but I think we can make something much simpler (probably local storage is enough).